plan/config.go

```diff
@@ -57,12 +59,19 @@ func ReadConfigFromEnviron() (*Config, error) {
 			behaviors[behavior.Name] = behavior
 		}
 	}
+
+	jsonReportPath := os.Getenv(jsonReportPathKey)
+	if jsonReportPath == "" {
+		jsonReportPath = "/crossdock/report.json"
```

Comment:
(Whether we want silent downgrade to regular console reporting or loud failure is up to you.)